sdk/drivers/lcd/Kconfig

```diff
@@ -172,4 +176,22 @@ config LCD_ILI9340_IFACE1_RGB565
 endchoice
 endif
 
+choice
```

Comment:
This configuration depends on the board.
So could you please change to define it in sdk/bsp/board/spresense/Kconfig.

Reply:
Ok, I'll fix this.